js/src/util/index.js

```diff
@@ -203,5 +219,6 @@ export {
   noop,
   reflow,
   getjQuery,
-  onDOMContentLoaded
+  onDOMContentLoaded,
```

Comment:
Is the `onDOMContentLoaded` export still used?

Reply:
Not that I'm aware of. I kept it like that because there is already a spec for it and to keep it testable. Also I wasn't sure if there would be some use for it in the future.

Reply:
As the last resort we could rename this to `_onDOMContentLoaded`, but since we test jQuery already, testing these individually might not be needed after all.

Reply:
It seems a better approach

Reply:
Actually, NVM my comment. Util isn't exposed in our public API https://github.com/twbs/bootstrap/blob/main/js/index.umd.js

So, I'd say @alpadev you can go ahead with this PR and ping us when it's ready. 👍